src/replication.c

```diff
@@ -3723,7 +3723,7 @@ int cancelReplicationHandshake(int reconnect) {
 }
 
 /* Set replication to the specified primary address and port. */
-void replicationSetPrimary(char *ip, int port) {
+void replicationSetPrimary(char *ip, int port, int try_psync) {
```

Comment:
Nit. I feel that `full_sync_required` is a better name?
```suggestion
void replicationSetPrimary(char *ip, int port, int full_sync_required) {
```